VideoLocker/res/layout/fragment_course_outline.xml

```diff
@@ -4,25 +4,26 @@
     android:layout_height="match_parent"
     android:background="@color/white"
     android:orientation="vertical"
-    android:splitMotionEvents="false" >
+    android:splitMotionEvents="false">
 
     <ListView
         android:id="@+id/outline_list"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        android:choiceMode="singleChoice"
         android:divider="@null"
-        android:dividerHeight="0dp"
-        android:listSelector="@drawable/course_list_item_selector"/>
```

Comment:
I don't think the `listSelector` should be removed here. There should be a separate `selector` drawable for the row background.